colour/quality/ssi.py

```diff
@@ -56,6 +58,8 @@ def spectral_similarity_index(
         Test spectral distribution.
     sd_reference
         Reference spectral distribution.
+    continuous
```

Comment:
Should we call it `round_output`, `rounded_output` rather than `continuous`? Seems more explicit!

Reply:
Done